RFS/src/main/java/org/opensearch/migrations/bulkload/workcoordination/WorkCoordinatorFactory.java

```diff
@@ -23,6 +24,8 @@ public OpenSearchWorkCoordinator get(
         ) {
         if (VersionMatchers.isOS_1_X.test(version) || VersionMatchers.isOS_2_X.test(version)) {
             return new OpenSearchWorkCoordinator_OS_2_11(httpClient, tolerableClientServerClockDifferenceSeconds, workerId);
+        } else if (VersionMatchers.isES_6_X.test(version)) {
```

Comment:
Can we consolidate down to a single constructor, or at least use overrides so we don't duplicate these branch statements 3 times?